tests.py

```diff
@@ -27,9 +30,14 @@ def run_vsm(self, name, input_data, expected_output, use_initial=True):
 
         process = Popen(cmd, stdin=PIPE, stdout=PIPE)
 
-        output, _ = process.communicate(input=input_data.encode(), timeout=2)
-
-        self.assertEqual(output.decode() , expected_output)
+        output, _ = process.communicate(input=input_data.encode(), timeout=5)
```

Comment:
And as for the expected output I thought it was fine just to print the state, ill update it to check the state thingy.